lib/src/main/java/com/github/omadahealth/lollipin/lib/managers/AppLockImpl.java

```diff
@@ -371,6 +376,24 @@ public boolean shouldLockSceen(Activity activity) {
         return true;
     }
 
+    @Override
+    public int getDefaultMaxLoginAttemptsAllowed() {
```

Comment:
I'm pretty sure we don't need this interface - we should be able to handle this ourselves outside of the library in our `onPinFailure` method.